Dockerfile.test

```diff
@@ -0,0 +1,26 @@
+ARG DOCKER_IMAGE
+
+FROM $DOCKER_IMAGE
```

Comment:
Hm.. I'm personally never a fan of an _empty_ default for `FROM`; ideally, it would have a sensible default, so that I can build the image successfully (`docker build ..`) without having to specify a `--build-arg`.

That will be a bit tricky in this case though

Reply:
Now that buildkit will be more used overall, perhaps we could still include the docker-py image as a build-stage in the main Dockerfile; buildkit will _skip_ that stage if it's not needed, and only build if the `--target` actually needs the build stage.

Doing so may make the `FROM xyz` easier to do.


@shin- @cpuguy83 @tonistiigi any suggestions?